src/options.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'https://esm.sh/react@18.2.0';
+/** @jsxImportSource https://esm.sh/v128/react@18.2.0 */
```

Comment:
Specifying import maps only in deno.jsonc does not change the compiled result. Also adding comment in tsx will be same hash.

```json
"options.js": "997528c14aecdc6d4376186bf2e357f58aba43c8b38afaf648462169c55ad767",
```

But writing as these pragma in each file changed it.

```json
"options.js": "0497f6529f2cd2d4ca3d5b9e0db81a12906544ea5f0024ba0ff58bdbd2a2bfff",
"options.js": "c2244bc109084e2a7febf9f866e07ec3d93c28a137492c77a6966c55e1b2eb74",
```